system-tests/projects/e2e-with-mount-import/cypress.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  e2e: {
```

Comment:
Instead of adding a new project, couldn't we use an existing one that has CT and E2E and just update the `support/e2e.js` to import `cypress/react`? Just to minimize the number of example projects.

Reply:
Moved to `component-tests` project, keeping the system-test project bloat down is good with me